### DISCUSSION TITLE
Adding Kotlin modules

### DISCUSSION THREAD

**Discussion by VadymIgnatiev** [matkoch/resharper-sampleplugin#89] [Q&A]
Can I connect a module written in Kotlin? In my case, it should make requests to external APIs into .net project(your plugin Template for ReSharper and Rider), and if so, how to do this, because I didn’t find information on this topic in your documentation?

**Comment by ForNeVeR** [comment]
I don't understand what exactly you mean by "connect a module", but Rider plugins support JVM dependencies (a most common form of that is Maven dependencies, though you can access anything you can add via Gradle, including local `.jar` files and whatnot).

**Comment by ErrorHQ** [comment]


**Reply by ForNeVeR** [reply]
What the actual heck. Is it a GPT-generated answer? It makes no sense!

"In your .NET project, add a reference to the Kotlin module or library" WAT
